source/s3_client.c

```diff
@@ -752,6 +752,14 @@ static struct aws_s3_meta_request *s_s3_client_meta_request_factory_default(
 
     /* Call the appropriate meta-request new function. */
     if (options->type == AWS_S3_META_REQUEST_TYPE_GET_OBJECT) {
+
+        /* If the initial request already has partNumber, the request is not splittable(?). Treat it as a Default
```

Comment:
This should be a backlog task somewhere.

Reply:
updated the Asana task, included the test is still needed in it.